src/Microsoft.TestPlatform.Common/Interfaces/Engine/ClientProtocol/IProxyExecutionManager.cs

```diff
@@ -21,6 +21,14 @@ public interface IProxyExecutionManager
     /// </summary>
     void Initialize(bool skipDefaultAdapters);
 
+    /// <summary>
+    /// 
+    /// </summary>
+    /// 
+    /// <param name="testRunCriteria"></param>
+    /// <param name="eventHandler"></param>
+    void InitializeTestRun(TestRunCriteria testRunCriteria, IInternalTestRunEventsHandler eventHandler);
```

Comment:
It seems really weird to me to have a method of a public interface having as parameter some `IInternal` stuff... Looks like a code smell to me.

Reply:
Agreed, but this interface is out there already and from the looks of it it is used on other public methods too. Sadly there's not much we can do here.

Reply:
🤦